gio/sys/src/manual.rs

```diff
@@ -10,19 +10,33 @@ pub use self::windows_constants::*;
 pub type GSocketFamily = libc::c_int;
 pub type GSocketMsgFlags = libc::c_int;
 
+#[cfg(target_family = "windows")]
+extern crate winapi as winapi_orig;
+
+#[cfg(target_family = "windows")]
+pub mod winapi {
+    pub use winapi_orig::shared::ws2def::AF_UNSPEC;
+    pub use winapi_orig::shared::ws2def::AF_UNIX;
+    pub use winapi_orig::shared::ws2def::AF_INET;
+    pub use winapi_orig::shared::ws2def::AF_INET6;
+    pub use winapi_orig::um::winsock2::MSG_OOB;
+    pub use winapi_orig::um::winsock2::MSG_PEEK;
+    pub use winapi_orig::um::winsock2::MSG_DONTROUTE;
+}
+
 #[cfg(target_family = "windows")]
 mod windows_constants {
     pub const G_SOCKET_FAMILY_INVALID: super::GSocketFamily =
-        self::winapi::shared::ws2def::AF_UNSPEC;
-    pub const G_SOCKET_FAMILY_UNIX: super::GSocketFamily = self::winapi::shared::ws2def::AF_UNIX;
-    pub const G_SOCKET_FAMILY_IPV4: super::GSocketFamily = self::winapi::shared::ws2def::AF_INET;
-    pub const G_SOCKET_FAMILY_IPV6: super::GSocketFamily = self::winapi::shared::ws2def::AF_INET6;
+        super::winapi::shared::ws2def::AF_UNSPEC;
+    pub const G_SOCKET_FAMILY_UNIX: super::GSocketFamily = super::winapi::shared::ws2def::AF_UNIX;
+    pub const G_SOCKET_FAMILY_IPV4: super::GSocketFamily = super::winapi::shared::ws2def::AF_INET;
+    pub const G_SOCKET_FAMILY_IPV6: super::GSocketFamily = super::winapi::shared::ws2def::AF_INET6;
 
     pub const G_SOCKET_MSG_NONE: super::GSocketMsgFlags = 0;
-    pub const G_SOCKET_MSG_OOB: super::GSocketMsgFlags = self::winapi::um::winsock2::MSG_OOB;
-    pub const G_SOCKET_MSG_PEEK: super::GSocketMsgFlags = self::winapi::um::winsock2::MSG_PEEK;
+    pub const G_SOCKET_MSG_OOB: super::GSocketMsgFlags = super::winapi::um::winsock2::MSG_OOB;
+    pub const G_SOCKET_MSG_PEEK: super::GSocketMsgFlags = super::winapi::um::winsock2::MSG_PEEK;
     pub const G_SOCKET_MSG_DONTROUTE: super::GSocketMsgFlags =
-        self::winapi::um::winsock2::MSG_DONTROUTE;
```

Comment:
Instead of all these changes, you can also keep the old code and replace `self::winapi::...` with `winapi::...`.

That's the only problem here :)

Reply:
Indeed! fixed it, thanks